code/src/components/CompletedToDos.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useNavigate } from 'react-router'
```

Comment:
Did useNavigate solve your problem with the pages that you had? What does it do? Navigate to the page that you want to go to?

Reply:
useNavigate is just another version of useHistory. I guess, navigate is a newer approach than history and to me it also makes more sense to call this method navigate. 
But it has nothign to do with the routes problem I had. This was solved by making an addition to the netlify.tomI